python/django/performance/access-foreign-keys.yaml

```diff
@@ -1,6 +1,8 @@
 rules:
   - id: access-foreign-keys
-    pattern: $X.user.id
+    patterns:
```

Comment:
maybe add a pattern-inside with a Django import to be relatively sure this is Django related?

```yaml
- pattern-inside: |
    from django.$MOD import $CLASS
    ...
```